### PR TITLE
Add Reachable/Unreachable Nodes and Delete Nodes Screens

### PR DESCRIPTION
**Pull request by germanferrero**
This PR is rebased with this other one https://github.com/libremesh/lime-app/pull/309, checkout the latter first.

Adds Reachable/Unreachable Screen and Delete Nodes from [this proposal](https://github.com/libremesh/lime-app/issues/308).

Storybook delete nodes: https://germanferrero.github.io/lime-app/?path=/story/containers-remove-nodes--delete-nodes-page

Storybook reachable nodes: https://germanferrero.github.io/lime-app/?path=/story/containers-reachablenodes--reachable-nodes-page

Basically one volatile shared state db (as usual) is used to know the list of connected nodes and one persistent shared state db (new feature) is used to know the complete list of network nodes.

### CODE REVIEW

**Comment by nicopace** [comment]
Is there a storybook place to check into this? thanks!

**Comment by germanferrero** [comment]
> Is there a storybook place to check into this? thanks!

Sure, thanks for asking: https://germanferrero.github.io/lime-app/?path=/story/containers-network-nodes--network-nodes-page

**Comment by nicopace** [comment]
in the ["Delete Nodes Page"](https://germanferrero.github.io/lime-app/?path=/story/containers-network-nodes--delete-nodes-page) the interaction in the screen appears when you select at least one node. if no node is selected, no clear call to action is defined.

A way to fix it would be to have it present all the time, and in the place of how many are selected, instruct the user to select something.

**Comment by nicopace** [comment]
in the ["Delete Nodes Page"](https://germanferrero.github.io/lime-app/?path=/story/containers-network-nodes--delete-nodes-page) the interaction in the screen appears when you select at least one node. if no node is selected, no clear call to action is defined.

A way to fix it would be to have it present all the time, and in the place of how many are selected, instruct the user to select something.

**Comment by nicopace** [comment]
it is not clear what the use of the connected/disconnected nodes view is...
Would be useful for the review to have the usecase at hand.

Thanks,

**Comment by spiccinini** [comment]
This is related to this issue enhancement proposal https://github.com/libremesh/lime-packages/issues/867

**Review by nicopace — APPROVED** [review]
thanks for the contextualization!

**Comment by nicopace** [comment]
Do you need anything for this branch to be merged @germanferrero ?

**Comment by germanferrero** [comment]
> in the ["Delete Nodes Page"](https://germanferrero.github.io/lime-app/?path=/story/containers-network-nodes--delete-nodes-page) the interaction in the screen appears when you select at least one node. if no node is selected, no clear call to action is defined.

@nicopace I'm waiting to have a bit of time to address this

**Comment by spiccinini** [comment]
I have some feedback:
Make the nodes clickable (to go to each node lime-app page), this may be done using a "three dots"  with a subemenu?

Add more information to each node in the list. I am not sure how it is best to do it but maybe we can add a button to display more information and select maybe only one column of information at a time to be desplayed. I mean, have the ability to display the hostname and its ipv4, or the hostname and its ipv6 (or board, or firmware version). This idea I think that does not shows too much information but it is very useful at the same time.  Other way can be to use the "three dots" menu with "show ipv4", "show ipv6", "show ...", actions.
What do you think? too much work?

**Comment by germanferrero** [comment]
Included in #310
